Project-Aurora/Aurora-LeagueLogiLEDWrapper/Aurora-LeagueLogiLEDWrapper.vcxproj

```diff
@@ -22,32 +22,32 @@
     <ProjectGuid>{7A7C78E5-0307-4139-9625-401A5A56C51F}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>AuroraLeagueLogiLEDWrapper</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.17134.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v141</PlatformToolset>
+    <PlatformToolset>v142</PlatformToolset>
```

Comment:
Please revert the toolset update.

Reply:
Yeah sorry about that, not quite sure what happened there but I've reverted it now

Reply:
I've really messed something up here, sorry